code/game/objects/structures/prop.dm

```diff
@@ -1191,12 +1207,76 @@
 /obj/structure/prop/vehicle/crawler/crawler_cargo
 	icon_state = "crawler_cargo"
 
+/obj/structure/prop/vehicle/big_truck
+	name = "military truck"
+	desc = "A military truck, made for transporting equipment or personnel in bulk."
+	icon = 'icons/obj/vehicles/large_truck.dmi'
+	icon_state = "truck"
+	coverage = 80
+	bound_height = 128
+	bound_width = 128
+	resistance_flags = XENO_DAMAGEABLE
+
+/obj/structure/prop/vehicle/big_truck/Initialize(mapload)
+	. = ..()
+	setDir(dir)
```

Comment:
? doesnt mapload call this

Reply:
doesn't appear to, at the very least it doesn't if you admin spawn something